extra/modules/confiant-ad-quality/src/test/java/org/prebid/server/hooks/modules/com/confiant/adquality/core/BidsMapperTest.java

```diff
@@ -0,0 +1,77 @@
+package org.prebid.server.hooks.modules.com.confiant.adquality.core;
+
+import com.iab.openrtb.request.BidRequest;
+import com.iab.openrtb.request.Imp;
+import com.iab.openrtb.response.Bid;
+import com.iab.openrtb.response.SeatBid;
+import org.junit.Test;
+import org.prebid.server.auction.model.BidderResponse;
+import org.prebid.server.bidder.model.BidderBid;
+import org.prebid.server.bidder.model.BidderSeatBid;
+import org.prebid.server.hooks.modules.com.confiant.adquality.model.RedisBidResponseData;
+import org.prebid.server.hooks.modules.com.confiant.adquality.model.RedisBidsData;
+import org.prebid.server.proto.openrtb.ext.response.BidType;
+
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class BidsMapperTest {
+
+    @Test
+    public void shouldMapBidResponsesToRedisBids() {
+        // given
+        BidRequest bidRequest = BidRequest.builder()
```

Comment:
Please, use `final` keyword everywhere. Also, please, check for similar occurences.